terralab/.terralab-cli-config

```diff
@@ -1,13 +1,12 @@
 # Teaspoons service API URL
-TEASPOONS_API_URL=https://teaspoons.dsde-dev.broadinstitute.org
+TEASPOONS_API_URL=https://teaspoons.dsde-prod.broadinstitute.org
 
 # Port to use for local server (for auth)
 SERVER_PORT=10444
 
-# Oauth config stuff
-OAUTH_OPENID_CONFIGURATION_URI=https://terradevb2c.b2clogin.com/terradevb2c.onmicrosoft.com/b2c_1a_signup_signin_dev/v2.0/.well-known/openid-configuration
-OAUTH_CLIENT_ID=bbd07d43-01cb-4b69-8fd0-5746d9a5c9fe
+# B2C auth config
+OAUTH_OPENID_CONFIGURATION_URI=https://terraprodb2c.b2clogin.com/terraprodb2c.onmicrosoft.com/b2c_1a_signup_signin_tsps_prod/v2.0/.well-known/openid-configuration
+OAUTH_CLIENT_ID=4aefc4ac-d5be-4440-ae9a-aefe4ceb098c
```

Comment:
is this id just the one that already existed in prod?  So like when we want to test against dev or bees we would need to change this value to what it used to be?  I guess this is a good reason to start thinking about how to dynamically generate this file (even if its just like the [write-config.sh](https://github.com/DataBiosphere/terra-scientific-pipelines-service/blob/main/scripts/write-config.sh) script from teaspoons)

Reply:
yes, if we want to run the CLI against non-prod deployment of teaspoons we'll need to update these values (all 3: teaspoons api url, oauth uri, oauth clientId). our e2e test does this for just the teaspoons api url.

Reply:
ticket: [TSPS-454 Provide an easier way to connect the CLI to non-prod teaspoons](https://broadworkbench.atlassian.net/browse/TSPS-454?atlOrigin=eyJpIjoiMjRmYjU2OTdmNzc5NGFhODk2OGY5ZDU1MjQxYmE4M2MiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)